marklogic-client-api/src/main/java/com/marklogic/client/impl/SSLUtil.java

```diff
@@ -25,7 +28,12 @@
 public interface SSLUtil {
 
 	static X509TrustManager getDefaultTrustManager() {
-		return (X509TrustManager) getDefaultTrustManagers()[0];
+		X509TrustManager trustManager = (X509TrustManager) getDefaultTrustManagers()[0];
+		Logger logger = LoggerFactory.getLogger(SSLUtil.class);
+		if (logger.isDebugEnabled() && trustManager.getAcceptedIssuers() != null) {
```

Comment:
I moved this logging here as it's useful to see for debugging purposes regardless of when this method is called.